src/loading-spinner/Loading.tsx

```diff
@@ -0,0 +1,28 @@
+import { GlobalStyles } from "../styles";
+import BounceLoader from "react-spinners/BounceLoader";
+
+const override = {
+  display: 'block',
+  margin: '0 auto',
+  position: "fixed",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+};
+
+const Loading: React.FC = () => {
```

Comment:
Here is a general purpose loading spinner that anyone can use in any asynchronous loading phase. I saw one such place in a PR from @macfarlandian earlier. We should standardize on this as the loading spinner. (UI approved by @sychang)